compiler/one-cmds/onelib/constant.py

```diff
@@ -54,6 +54,7 @@ class CONSTANT:
          'replace channel-wise Mul/Add with DepthwiseConv2D'),
         ('remove_fakequant', 'remove FakeQuant ops'),
         ('remove_quantdequant', 'remove Quantize-Dequantize sequence'),
+        ('remove_redundant_quantize', 'remove subsequent Quantize ops'),
```

Comment:
This pass can remove a single redundant Quantize Op, so I think the below explanation would be better.

```suggestion
        ('remove_redundant_quantize', 'remove redundant Quantize ops'),
```